0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -194,39 +199,13 @@ Transport layer protocols are responsible for coordinating the different transfe
-
-For most cases, OTP will not be appropriate. However, it may make sense in the case of recurring microtransactions where a few lost transactions are not a big deal.
-
-<span class="show alert alert-warning">**TODO:** Need to figure out how OTP can detect dropped transfers and adjust the route accordingly.</span>
-
-#### Universal Transport Protocol (UTP)
-
-The Universal Transport Protocol (UTP) is the standard and recommended transport protocol. It sets up a cascading chain of escrowed transfers in order to ensure delivery of funds.
-
-The protocol flow is described in the [Interledger whitepaper](https://interledger.org/interledger.pdf).
-
-##### Error Recovery
-
-A UTP execution chain can be interrupted if a connector fails to deliver the condition fulfillment. In that case the sender will think that the transaction failed and retry it. While retrying, some connector will try to create a prepared transfer on a ledger which already knows the corresponding fulfillment, which will cause the request to fail. The connector will notice this and simply pass back the fulfillment without paying anything.
-
-In other words, the failed connector will lose money and some other connector will win money, but from the sender and recipient's point of view everything has executed normally.
-
-#### Atomic Transport Protocol (ATP)
-
-The Atomic Transport Protocol (ATP) is the most conservative, but also most complex of the standard Interledger transport protocols. In addition to the ledgers and connectors, it also involves a set of impartial notaries, which must be agreed upon by the sender, recipient and connectors involved in the transaction.
-
-Since each party chooses their own set of notaries to trust, there may not be any overlap and ATP could not be used in that case. Notaries may be selected using an automatic process or preselected by a group with standing agreements among the participants.
-
-The protocol flow is described in the [Interledger whitepaper](https://interledger.org/interledger.pdf).
+#### Privately Shared Key (PSK)
 
-##### As a Sub-Protocol
+<span class="show alert alert-warning">**TODO:** Describe PSK.</span>
 
-ATP can be used as a sub-protocol for part of a UTP transaction. This happens when a connector decides to create an ATP transfer as the next hop. It takes on both the risk of being unable to pass on the receipt and the risk of the notaries failing. This makes sense to do if the next connector quotes a better price for an ATP transfer than for a UTP transfer.
+#### Interledger Payment Request (IPR)
 
-Similarly, an ATP transaction can turn into a UTP transaction when a connector decides to use UTP for the next hop. It takes on the risk of not being able to pass on the receipt to the notaries. This makes sense if ATP is not available.
+<span class="show alert alert-warning">**TODO:** Describe IPR.</span>
```

Comment:
We'll definitely link it, but the previous version of this doc had a short paragraph for each protocol / component, which was quite useful I thought.